app/routes.php

```diff
@@ -9,7 +9,7 @@
     $app->get('/admin', 'AdminController');
     $app->get('/register', 'RegisterController');
     $app->get('/addapplicant', 'addApplicantController');
-    $app->get('/applicants', 'DisplayApplicantsController');
+    $app->get('/applicants[/{page}]', 'DisplayApplicantsController');
```

Comment:
the optional/dynamic portion of the URL doesn't get used, and if you manually go to /applicants/2 for example the page is just broken.

Suggest removing this and just making the route for /applicants